mkdocs.yml

```diff
@@ -43,4 +43,5 @@ copyright: Copyright &copy; 2014 <a href="https://twitter.com/_tomchristie">Tom
 google_analytics: ['UA-27795084-5', 'mkdocs.org']
 
 plugins:
-    - search
+    - search:
+        prebuild_index: python
```

Comment:
I realize having this enabled helps with testing, but I think perhaps we should not have this enabled for our own documentation. According to my testing (with the node implementation some months back), the size of our documentation does not justify the extra size to the index by prebuilding. That is, unless you can provide some clear benefits.

Reply:
absolutely, I included it to show how the `prebuild_index` looks like now, I'll remove it 👍